### ISSUE TITLE
prepare help overlay

### ISSUE THREAD

**Issue by thvitt** [faustedition/faust-web#25]
Idea was to add a help overlay (based on chardin.js) to each of the “work” pages that should be triggered by the help button. The help overlay should explain the most important items of the user interface, offer stuff that is needed in-place like a legend, and offer a link to the corresponding part of the “about the edition” text.

* [X] add support for HTML tooltips to chardin.js
* [ ] add support for ‘global’ overlay content to chardin.js
* [ ] add example support for help overlay to a rather static page
* [ ] add support for help overlay to generated content

**Comment by gerritbruening** [comment]
Will this help overlay also serve as "legend", or do we need a legend, e.g., in the bar diagram, that is visible without prior interaction?

**Comment by thvitt** [comment]
won’t manage to do that in time. 

We’ll fill the help page with a linked introduction and adjust the help button for each page.

**Comment by gerritbruening** [comment]
So we proceed with #494?

**Comment by thvitt** [comment]
yes, and maybe add some usage-oriented stuff on main bar and document viewer, if we don’t have that already
